containers/images/pulp/Containerfile.core.j2

```diff
@@ -102,6 +103,10 @@ RUN pip install {{ plugin }}
 {% endfor %}
 {% endif %}
 
+RUN PULP_STATIC_ROOT=/var/lib/operator/static/ PULP_CONTENT_ORIGIN=localhost \
+	/usr/local/bin/pulpcore-manager collectstatic --clear --noinput --link && \
+	chown -R 1000 /var/lib/operator/static/
```

Comment:
it works!
![image](https://user-images.githubusercontent.com/17153022/190506320-6cebcdea-138b-4b16-b2b5-7a744622b2ec.png)
@git-hyagi++